src/main/java/org/olhab/springboot/service/TodoService.java

```diff
@@ -0,0 +1,16 @@
+package org.olhab.springboot.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.olhab.springboot.dto.todo.TodoCreateDto;
+import org.olhab.springboot.dto.todo.TodoResponseDto;
+import org.olhab.springboot.dto.todo.TodoUpdateDto;
+
+public interface TodoService {
+    TodoResponseDto save(TodoCreateDto todoCreateDto);
+
+    TodoResponseDto update(TodoUpdateDto todoCreateDto, Long id) throws JsonProcessingException;
```

Comment:
JsonProcessingException should be handled in the TodoServiceImpl
```suggestion
    TodoResponseDto update(TodoUpdateDto todoCreateDto, Long id);
```